src/extension/importer/core/src/main/java/org/geoserver/importer/DataFormat.java

```diff
@@ -114,7 +115,7 @@ protected String relativeDataFileURL(String url, Catalog catalog) {
             return url;
         }
         File baseDirectory = catalog.getResourceLoader().getBaseDirectory();
-        File f = Files.url(baseDirectory, url);
+        File f = Resources.find(Resources.fromURL(Files.asResource(baseDirectory), url), true);
```

Comment:
There should be a static constant of some sort constant or method for better readability....
Looks like we are doing a round trip from Path --> File --> Resource --> File above?

Hold on, this method is used by data sources to access data, was trying to do this with Files (and only use Resources when working with configuration information).

Reply:
I've run as many changes as possible through automatic refactoring, some of the results are less than appealing.

Reply:
Looking at it again, it seems like an equivalent of what the previous code was doing, just in a more evident way.

Reply:
Here is a PR (against this branch) reverting the one commit, and removing the deprecation on Files.url https://github.com/aaime/geoserver/pull/9